### DISCUSSION TITLE
2022.07.26 MSF Delegates Meeting 19:30 CET

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/MSF-Delegates#26] [General]
- Volunteers met in the OMI weekly meeting channel with ongoing discussions taking place @ [#msf-delegates](https://discord.com/channels/770382203782692945/1000781076463112234).
- Deep dove into the MSF member portal
  - Lyuma and others suggested MSF Standards Registry subgroup matches well with OMI
    - OMI has "joined" that subgroup (for now under the "OMI Shared" account)
    - Its kickoff meeting is tomorrow Wednesday 7AM PDT
    - humbletim planning to attend / live stream in OMI discord for interested community members
  - Gathered delegate portal account applications for batch approval
    - at first Khronos was receptive towards using our own email addresses
    - UPDATE: we've since received push back (so may need to reconsider one of the alternate strategies discussed)
- Robert suggested using OMI Google Drive where community access is needed to large assets (such as MSF meeting recordings)
- Funksoup volunteered to help rough draft a mission statement capturing MSF delegates group vision
- LMB suggested we look into mapping MSF emerging topics to OMI community
    re: <img src=https://user-images.githubusercontent.com/94753/181107427-fa3f838c-b261-448c-bcb7-f8ae7d74c805.png  width=128 />
- Jin encourages everyone to leverage documentation
  - a later question emerged of where to put things (eg: a new MSF github repo?)
  - for now consensus is to coordinate using meta discussions in main `omigroup` repo
- A. suggested we do a Miro jam session (and it became our primary agenda item for next week)
- New `@MSF Delegate` role available in discord (thanks Jesse!)
  - A. suggested we wire up to #roles channel for discord community opt-in
- we confirmed current meeting time works for now and planned to continue same time next week.
- humbletim to create next week's Discord Event listing

... if these notes have overlooked anything please add via comment below.

(tentative agenda for next week)
### MSF Delegate meeting Tuesday August 2nd 1930 CET
  - (if this time continues to work) should we add to OMI calendar?
  - follow-up on MSF delegate portal access
  - any relevant highlights from MSF meetings
  - Miro jam session to brainstorm and diagram ecosystem/participation/community strategies
  - ...

**Comment by pcsmgnt** [comment]
Thank you Tim. Great notes! 
1. Do you have a link to the OMI Google Drive?
2. We also agreed to maintain content within the main OMI git repo for now
3. Should we add to OMI calendar? **_Yes_**
4. We also discussed whether we could set up webhooks for the MSF-delegate channel for key announcements? @mrmetaverse

**Reply by mrmetaverse** [reply]
1. https://drive.google.com/drive/folders/1YWB9v3rDzXgMSoAgCCThnwB3lO_GyN_8

2. We don't use Google drive though, and should refrain from doing so. We put all our files in GitHub folders. Adding tools has historically added complication and makes it more difficult to find things and manage access and security. In the past if I have shared a Google drive link it was out of immediate necessity due to being on mobile or something like that. 

3. Happy to add to the calendar. When do you meet? 

4. Happy to set up a webhook. I assume we can have it trigger events on a keyword. Otherwise, I'd suggest a MSF repo so I can federate tickets and announcements that way, like we have done with every other repo thus far. 


Thanks! Cc: @pcsmgnt @humbletim

**Comment by pcsmgnt** [comment]
@mrmetaverse 
1. Noted on the Google Drive
2. We meet on Tuesdays at 17:30 PM UTC. 
3. We have yet to decide the keywords, but key events such and polls, meetings etc would be a good start.

Thoughts?

**Reply by mrmetaverse** [reply]
I think a repo is the best way for you to be able to do all of that and have it sync with the channel. Plus I have a master repo overseeing all repos so I don't Actually see any benefit to not having a omigroup/MSFdelegates repo. 

Can you all come to consensus on if you want a repo?

**Reply by humbletim** [reply]
re: meeting time -- suggest waiting one more iteration before locking in current time (to afford async feedback time for those who couldn't attend first meeting to chime in..)

re: git repo -- one concern robert expressed at the meeting was that it might be challenging to keep track across all the different repos; maybe that aspect could be solved by setting up some "interest set" project views?
